setup.py

```diff
@@ -23,5 +26,5 @@
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.3',
-    ]
-)
+        ]
```

Comment:
Hey, for sude this adds no benefit. I just like it better because it keeps +4 spaces from the original indent. But I'm really happy to revert.